suite-native/accounts/src/components/AddAccountsButton.tsx

```diff
@@ -38,7 +36,7 @@ export const AddAccountButton = ({ flowType }: AddAccountButtonProps) => {
     };
 
     const navigateToAddCoinAccount = () => {
-        if (isAddCoinAccountEnabled) {
+        if (isDeviceConnectEnabled) {
```

Comment:
Is not this condition redundant now? If `isDeviceConnectEnabled` === `false` user should not be able to execute this function anyway.

Reply:
that is a failsafe :) but yes, he should not

Reply:
I would vote for removing it then to make the code as simple as possible. But I will leave this up to you. Approving :)

Reply:
updated